camera_matrix: [743.023418,0.000000,329.117496,0.000000,745.126083,235.748102,0.000000,0.000000,1.000000]
dist_coeffs: [0.182276,-0.533582,0.000520,-0.001682,0.000000]